src/js/keyboardInput.js

```diff
@@ -178,7 +183,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 method: "on",
                 args: ["focusin.closeTooltip", "{that}.tooltip.close"],
                 priority: 1
-            }
+            },
+            "shiftLatchChange.toggleShiftLatched": "{that}.toggleShiftLatched"
```

Comment:
See the line 188 below, you probably wanna move this listener out of the block of the tooltip handling.